RadarSDK/RadarSettings.h

```diff
@@ -36,9 +36,9 @@ NS_ASSUME_NONNULL_BEGIN
 + (void)setListenToServerTrackingOptions:(BOOL)listenToServerTrackingOptions;
 + (RadarTrackingOptions *_Nullable)trackingOptions;
 + (void)setTrackingOptions:(RadarTrackingOptions *_Nonnull)options;
-+ (RadarTrackingOptions *_Nullable)fallbackTrackingOptions;
-+ (void)setFallbackTrackingOptions:(RadarTrackingOptions *_Nonnull)options;
-+ (void)revertToFallbackTrackingOptions;
++ (RadarTrackingOptions *_Nullable)remoteTrackingOptions;
```

Comment:
I think we should call this `clientTrackingOptions` to be explicit about where these options are coming from